### ISSUE TITLE
scipy.signal.step

### ISSUE THREAD

**Issue by Godjka** [scipy/scipy#12116]
**Is your feature request related to a problem? Please describe.**
The current function signal.step() simulates for a unitary step. Unfortunately, it's not possible to change the size of the step. The system that I'm trying to simulate, the unitary step is a value that is too big for the system. 

**Describe the solution you'd like**
Would it be possible to add another argument where the user can specify the amplitude of the step? Alternatively, add on the documentation the parameter that I should edit to manually modify the amplitude. 

**Describe alternatives you've considered**
To use the scipy.integrate.odeint() function instead.

Thank you.

**Comment by WarrenWeckesser** [comment]
`step` computes the step response of a linear system, so if you, say, double the amplitude of the step, the output amplitude doubles.  So you can simply multiply the output of `step` by whatever step amplitude that you are interested in.

**Comment by Godjka** [comment]
Yes, since it's a linear system I can divide by the step amplitude to get more realistic values.
I just thought that it would be interesting to have that option, or at least to be able to manually modify it since it's possible in Matlab.

Thank you for the reply!
